ui/components/src/test/properties/renderers/value/URIPropertyValueRenderer.test.tsx

```diff
@@ -0,0 +1,115 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { expect } from "chai";
+import * as React from "react";
+import { Id64 } from "@bentley/bentleyjs-core";
+import { render } from "@testing-library/react";
+import { PropertyValueRendererContext } from "../../../../ui-components/properties/ValueRendererManager";
+import TestUtils from "../../../TestUtils";
+import { URIPropertyValueRenderer } from "../../../../ui-components/properties/renderers/value/URIPropertyValueRenderer";
+import { PropertyRecord } from "@bentley/ui-abstract/lib/ui-abstract/properties/Record";
+import { DEFAULT_LINKS_HANDLER } from "../../../../ui-components/properties/renderers/value/PrimitivePropertyValueRenderer";
+
+describe("URIPropertyValueRenderer", () => {
+
+  describe("render", () => {
+    it("renders URI property wrapped in an anchored tag from display value", () => {
+      const renderer = new URIPropertyValueRenderer();
+      const property = TestUtils.createURIProperty("Category", "Value", "Test Uri Value: pw:\\wsp-aus-pw.bentley.com:wsp-aus-pw-10\Documents\Southern Program Alliance");
+
+      const element = renderer.render(property);
+      const elementRender = render(<>{element}</>);
+
+      expect(elementRender.container.getElementsByClassName("core-underlined-button")[0].textContent).to.be.eq("Test Uri Value: pw:\\wsp-aus-pw.bentley.com:wsp-aus-pw-10\Documents\Southern Program Alliance");
+    });
+
+    it("renders URI property wrapped in an anchored tag if links are specified in the PropertyRecord", () => {
+      const renderer = new URIPropertyValueRenderer();
+      const property: PropertyRecord = TestUtils.createURIProperty("Category", "Value", "Test www.test.com");
+
+      property.links = DEFAULT_LINKS_HANDLER;
```

Comment:
Let's not use `DEFAULT_LINKS_HANDLER`. Using some custom implementation would be better to confirm it's actually using what we provide.